src/plugins/plugin.legend.js

```diff
@@ -408,19 +417,30 @@ var Legend = Element.extend({
 				}
 			};
 
+			var alignmentOffset = function(dimension, blockSize) {
+				if (opts.align === 'start') {
+					return labelOpts.padding;
+				} else if (opts.align === 'end') {
+					return dimension - blockSize;
+				}
+				// default to center
+				return (dimension - blockSize + labelOpts.padding) / 2;
+			};
+
 			// Horizontal
 			var isHorizontal = me.isHorizontal();
+			var line = 0;
```

Comment:
Yes, that was the intention - with the additional uses, the thinking was to replace with a descriptive variable name over a hardcoded "magic number".  Admittedly I could have chosen a _more_ descriptive name (`firstLine` perhaps?), and the number isn't _that_ magical.  Also, byte-shaving is somewhat new to me.  In any case, I don't feel that strongly and will revert if so desired.

Reply:
If the line is not going to change, then yes, let's keep `0` everywhere. You can add a comment if you feel that it needs to be explained why we are using `0` instead of whatever other `index`, but I don't think `line` or `firstLine` explain why, neither help to refactor.

Reply:
👍